mixed_requests/test_mixed.py

```diff
@@ -60,6 +60,10 @@ def create_servers(self):
         port = tempesta.upstream_port_start_from()
         self.servers = [control.Nginx(listen_port=port)]
 
+    @unittest.expectedFailure
```

Comment:
It's good to add corresponding issue number for the failure.

Reply:
Added description